### DISCUSSION TITLE
Migration to GitHub Issues

### DISCUSSION THREAD

**Discussion by radioactiveman** [audacious-media-player/audacious#125] [Announcements]
As mentioned in our previous [announcement](https://github.com/orgs/audacious-media-player/discussions/65 
), we intend to transition away from Redmine to GitHub.

After enabling (GitHub) discussions here the next step will affect our bug tracker. The plan is to migrate all previous bug reports with the tool [redmine2github](https://github.com/iqss/redmine2github). I have already tested it successfully in a smaller scale and want to start with the actual migration in April.

Important thing to know is that GitHub will send notifications for every issue and comment to all persons who watch our GitHub repo.
To avoid receiving hundreds of emails please adjust your notification settings for the main [audacious project](https://github.com/audacious-media-player/audacious).

Either configure custom settings like in the screenshot below and uncheck issues or simply disable notifications temporarily.
I will keep you updated about the current migration status here.

![grafik](https://github.com/audacious-media-player/audacious/assets/1301412/d55a6310-008d-4d97-b3f9-fb0bdc1c54f6)

**Comment by radioactiveman** [comment]
Current status:

- [x] Create a separate GitHub user for the migration
- [x] Enable GitHub issues
- [x] Run the actual migration
- [x] Copy attachments of open issues
- [x] Clean-up the issue labels and milestones
- [x] Add templates for issues / feature requests
- [x] Update the bug tracker links on our website and in Git
- [ ] Retire Redmine and make it read-only

**Comment by radioactiveman** [comment]
Unfortunately an unexpected issue occurred:

My plan was to migrate all Redmine issues to GitHub so they have the same issue number to keep references from commits or changelogs working. However GitHub uses the same counter for Issues, Pull Requests and Discussions. Our ["first" issue](https://github.com/audacious-media-player/audacious/issues?q=is%3Aissue+is%3Aclosed) here on GitHub starts now with number 135...

@jlindgren90: How should we proceed now? Does it make sense to you to still migrate all issues from Redmine? Or should we just focus on the open ones?

**Reply by jlindgren90** [reply]
I'd vote for migrating all of them as planned. Then we have a backup (of sorts) of the Redmine data even if the numbers don't match up.

**Reply by birdie-github** [reply]
You could create a new project, migrate only issues and then copy the remaining pull requests and discussions from this one and then deprecate this one.

**Reply by jlindgren90** [reply]
I don't like that, then we lose all the pull request history.

**Reply by birdie-github** [reply]
```patch
-You could create a new project, migrate only issues and then copy the remaining pull requests and discussions from this one and then deprecate this one.
+You could create a new project, migrate only issues and leave them be in that project.
+And you could then disable the Issues feature for the audacious and audacious-plugins projects altogether.
```
:-)
